Makefile

```diff
@@ -65,3 +65,13 @@ deb: build
 		--description "Boulder is an ACME-compatible X.509 Certificate Authority" \
 		--maintainer "$(MAINTAINER)" \
 		test/config/ sa/_db data/ $(OBJECTS) bin/ct-test-srv
+
+tar: build
```

Comment:
Make sure you add `tar` to the list of `.PHONY` targets up on line 28. You'll have to rebase or merge in the change that Matthew just landed to avoid a merge conflict.